### DISCUSSION TITLE
cnfirming match

### DISCUSSION THREAD

**Discussion by ghostrunner17** [ValveSoftware/Dota2-Gameplay#22360] [General]
10 days have already past and everytime i stuck for 20 mins in confirming match bug in euw and valve still doesnt do anything..

**Comment by norma44shah** [comment]
I'm sorry to hear you're experiencing this frustrating issue with the "Confirming match..." bug in EUW. It seems like many players are facing the same problem, and it's been ongoing for quite some time1. Unfortunately, Valve hasn't addressed it yet.

Have you tried changing your matchmaking region to see if that helps? Some players have found temporary relief by switching to a different server [TellCulver](https://www.tellculvers.cc)
